snapcraft/internal/lxd/_project.py

```diff
@@ -151,6 +155,30 @@ def _remote_mount(self, destination, source):
             'You can run the following command to enable it:\n'
             'echo Y | sudo tee /sys/module/fuse/parameters/userns_mounts')
 
+    def _setup_user(self):
```

Comment:
This will setup a user even if snapcraft happens to be run as sudo. Is that desired?

Reply:
The check is at the calling site. I'll move it around.

Reply:
A comment here would be helpful.